docker-app/qfieldcloud/core/templates/admin/job_change_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'admin/change_form.html' %}
+
+{% block submit_buttons_bottom %}
+  {{ block.super }}
+
+  <div class="submit-row">
+    {% if original %}
+        <a href="{% url 'admin:rerun_job' original.pk %}" class="button">Re-run</a>
```

Comment:
```suggestion
        <a href="{% url 'admin:rerun_job' original.pk %}" class="button">{% trans 'Re-run job' %}</a>
```

NOTE this needs the`i18n` to be imported in this template.

Reply:
done